bower.json

```diff
@@ -25,7 +25,7 @@
     "angular-smart-table": "~2.1.1",
     "angular-chart.js": "~0.7.2",
     "angular-cookies": "~1.4.3",
-    "lodash": "4.6.1",
+    "lodash": "~4.6.1",
```

Comment:
The build is now failing on [Travis](https://travis-ci.org/esaude/esaude-emr-poc/builds/118979971#L724) with:

```sh
bower ECONFLICT     Unable to find suitable version for lodash
```

I can confirm that when I run `bower install` locally I get prompted to manually select a `lodash` version:

```sh
Unable to find a suitable version for lodash, please choose one:
    1) lodash#~3.10.1 which resolved to 3.10.1 and is required by angular-datepicker#1.0.20
    2) lodash#~4.6.1 which resolved to 4.6.1 and is required by ui-prototypes
    3) lodash#4.6.1 which resolved to 4.6.1
```
I can't commit to this PR, but you can fix the build on Travis by making the following change:

```diff
diff --git a/.travis.yml b/.travis.yml
index 6fb860d..10a2f1b 100755
--- a/.travis.yml
+++ b/.travis.yml
@@ -6,7 +6,7 @@ before_install:
     - sh -e /etc/init.d/xvfb start
 before_script:
     - npm install -g bower grunt-cli codacy-coverage coveralls
-    - bower install
+    - bower install --force-latest
 after_script:
     - cat coverage/**/lcov.info | codacy-coverage
     - cat coverage/**/lcov.info | coveralls
```